shared/utils/routeHelpers.js

```diff
@@ -61,3 +61,5 @@ export function settings(): string {
 export function groupSettings(): string {
   return `/settings/groups`;
 }
+
+export const URL_REGEX = /^([0-9a-zA-Z-_~]*)-([a-zA-Z0-9]{10,15})$/;
```

Comment:
```suggestion
export const SLUG_URL_REGEX = /^([0-9a-zA-Z-_~]*)-([a-zA-Z0-9]{10,15})$/;
```

Might be clearer in the future now it's no longer within the context of documents specifically.